pkg/controller/pv.go

```diff
@@ -242,12 +244,20 @@ func (r *ReconcilePV) Recover(ctx context.Context) error {
 		var diskName string
 		diskName, err = azureutils.GetDiskName(pv.Spec.CSI.VolumeHandle)
 		if err != nil {
-			return err
+			w.Logger().Errorf(err, "failed to recover pv (%s)", pv.Name)
```

Comment:
np: please spell out `PersistentVolume` instead of abbreviated it. Same applies below.

Reply:
let's also log the error for debugging.

Reply:
Done.